left:
  camera_model: pinhole
  distortion_coeffs: [-0.289535816731999, 0.07993416587215198, 0.00012090041116053658, 3.212578238912377e-05]
  distortion_model: radtan
  intrinsics: [458.9999395464594, 457.7172227907516, 366.43661081346886, 249.39257294828]
  resolution: [752, 480]
  topic: /cam0/image_raw

right:
  camera_model: pinhole
  distortion_coeffs: [-0.28649216573519964, 0.07680040156229297, -0.00012439375196757852, -2.9930914938629677e-05]
  distortion_model: radtan
  intrinsics: [457.748146447752, 456.39320018537245, 379.4358081256208, 256.2865675829975]
  resolution: [752, 480]
  topic: /cam1/image_raw

T_left2right:
  [ 0.9999976735038351, 0.0021496301253002264, -0.000179101167272723, -0.10999930262275599,
    -0.0021469187858294894, 0.999900139038914, 0.013967916439968931, 0.0003992528504798459,
   0.00020910913602506244, -0.013967499428004416, 0.9999024278563806, -0.0005123105626500769,
    0.00000000000000000,  0.00000000000000000,  0.00000000000000000,  1.00000000000000000]

pub_match_image_topic: /feature_tracker/image
pub_feature_topic: /feature_tracker/points
frequence: 10
board_size: 1
tracker_size: 10
tracker_min_distance: 20
tracker_max_count: 100
equalize: true
pyramid_level: 3
win_size: 21
f_threshold: 1.0
show_tracker: true